docker-arm64/Dockerfile

```
@@ -54,10 +54,10 @@ ENV PATH="/go/bin:/usr/local/go/bin:${PATH}"
 
 # Install vault: https://github.com/hashicorp/vault/blob/main/CHANGELOG.md https://releases.hashicorp.com/vault
 RUN curl -LO https://releases.hashicorp.com/vault/$VAULT_VERSION/$VAULT_FILENAME \
-    && echo "${VAULT_CHECKSUM}  ${VAULT_FILENAME}" | sha256sum --check \
-    && unzip $VAULT_FILENAME \
+    && echo "${VAULT_CHECKSUM} ${VAULT_FILENAME}" | sha256sum --check \
+    && unzip $VAULT_FILENAME -d /tmp/vault \
     && rm $VAULT_FILENAME \
-    && mv vault /usr/bin/vault \
+    && mv /tmp/vault/vault /usr/bin/vault \
     && chmod +x /usr/bin/vault
```

Rather than unzipping the whole archive (with the `LICENSE` file we’re not interested in) in a temporary directory and move the file we’re interested in (`vault`) in its final destination in a second step, it’s possible to tell `unzip` to inflate only the file we’re interested in (`vault`) directly in the correct final directory in a single line.

```suggestion
    && unzip -o $VAULT_FILENAME -d /usr/bin -o vault \
    && rm $VAULT_FILENAME
```

---

Thanks! Will do this for all the docker files in a follow-up PR 🙇